### PR TITLE
Add option the check version of sgpt

### LINKED ISSUE #161: Check version of ShellGPT and warn user
Implement a version report for `sgpt` using `--version` option. 

Optional:
During every launch check if updates available (using PYPI or GitHub releases). Provide instructions to user how to update if there is a new version. This should be something efficient so we don't block a main thread during launch. Or check for update every X days.

### PR DESCRIPTION
**Pull request by Navidur1**
Add an option to check the version of `sgpt` using `--version` or `-v`.

Resolves #161 

image

### CODE REVIEW

File: sgpt/version.py
Comment (TheR1D) [inline sgpt/version.py:1]: Let's rename this file as `__version__.py`.

**Review by TheR1D — CHANGES_REQUESTED** [review]

**Comment by Navidur1** [comment]
@TheR1D fixed the previous suggestion! Let me know if there's anything else that needs to be fixed

File: pyproject.toml [hunk 1]
```diff
@@ -60,7 +60,7 @@ dev = [
 ]
 
 [tool.hatch.version]
-path = "sgpt/__init__.py"
+path = "sgpt/version.py"
```
Comment (TheR1D) [inline pyproject.toml:63]: Need to change the file name here as well.

**Review by TheR1D — CHANGES_REQUESTED** [review]

**Review by TheR1D — APPROVED** [review]

**Comment by TheR1D** [comment]
Thanks, merged.

**Pull request merged**